### ISSUE TITLE
Aqara multisensor offset 

### ISSUE THREAD

**Issue by b3nsk0** [ebaauw/homebridge-deconz#82]
Hi,

first of all, thank you very much for your efforts regarding the homebrige-hue and deconz theme, I appreciate your work!

Recently I moved from homebridge-hue to homebridge-deconz. I own an Aqara multisensor, with Homebridge-hue it was possible possible to set an offset for all the sensors (temperature, humidity and air pressure), now with homebridge-deconz I could only set an offset for the temperature in the Eve app. Will it be possible to bring back the option to have offset for the other two, especially humidity? Thank you very much!

**Comment by ebaauw** [comment]
>  Homebridge-hue it was possible possible to set an offset for all the sensors (temperature, humidity and air pressure)

I don't think so.  Homebridge Hue only exposes `offset` for ZHATemperature and ZHAThermostat `sensors` resources, not for ZHAHumidity nor for ZHAPressure.

To expose offset for humidity or air pressure, I would need to use different HomeKit characteristics, as the current _Offset_ has unit °C.  While technically doable, I'm not sure if this is the right way to go.  I tried to do dynamic configuration from HomeKit in Homebridge deCONZ, and that doesn't work out too well.  Also, you don't want regular HomeKit users / family members messing with your configuration, and admin-only access in HomeKit also doesn't work out too well.

I want to move the configuration settings (_Offset_, _Low Battery Threshold_, _Heartbeat_, _Log Level_, _Search_, _Unlock_, _Expose_, ...) to the Homebridge UI rather than exposing more of these to HomeKit.

**Comment by ebaauw** [comment]
Still need a way to provide settings per service instead of per accessory.  Note that an accessory might contain multiple _Tempereature Sensor_ services, each with a different offset.  So apart from being ugly, using a `temperatureOffset` vs `humidityOffset` on the accessory isn't going to cut it.
